docs/guide/docs/advanced/authentication.md

```diff
@@ -3,55 +3,119 @@ id: authentication
+3. **External User**: This method of authentication expects that you handle user authentication yourself and provide a JWT token to identify the user.
 
-We support Basic, SAML and LDAP for the Admin authentication.
+## Authentication Overview
 
-User Authentication is managed using JWT tokens
+There are 4 different type of authentication strategies currently supported: Basic, SAML, OAUTH2 and LDAP. You can also implement them multiple times (for example, you could have two different OAuth2 configurations for different workspaces).
 
-## Admin Authentication
+No matter what authentication strategy is used, they are all stored in the database. When you add a new strategy in the `botpress.config.json` file, upon server restart a new table called `strategy_STRATEGYID` will be created.
 
-There are 3 different type of authentication methods. By default, the `basic` method is used.
+When you give access to a user for a specific workspace, an entry is created in the table `workspace_users` with his role.
+
+If you have more than one authentication strategy, a menu will be displayed to pick a strategy. You can skip the menu and bookmark a specific strategy by changing the page URL: `/admin/login/STRATEGYID`.
+
+Moreover, you can access a specific workspace by using `/admin/login?workspaceId=WORKSPACEID`
+
+https://github.com/botpress/botpress/blob/master/src/bp/core/config/botpress.config.ts#L326
 
 ### Basic
 
-Users logs in with an Email / Password combination. Information is saved in the file `data/global/workspaces.json`, and they are salted for added security.
+Basic Authentication allows a user to log in with a simple username / password. The password is salted for added security.
 
 To create more accounts, visit the `Collaborators` tab on the Admin UI. Choose the role and enter the E-mail of your collaborator, then you will receive a random password. The user will need to pick a password after the first login.
 
-To enable this authentication method, set `pro.auth.strategy = 'basic'` in your `botpresss.config.json` file.
+Super Admins are able to reset the password of any user using the basic authentication.
+
+Here's a basic configuration example to set it up:
+
+```js
+{
+ "pro": {
+    "collaboratorsAuthStrategies": ["default"],
+  },
+  "authStrategies": {
+    "default": {
+      "type": "basic",
+      "label": "Default Auth",
+      "options": {}
+    }
+  }
+}
+```
+
+#### Additional Security
+
+There are additional options that can be configured when using that authentication strategy. Please refer to the [configuration file for more informations](https://github.com/botpress/botpress/blob/master/src/bp/core/config/botpress.config.ts#L350) :
+
+- maxLoginAttempt: Max number of tries allowed before locking out the user
+- lockoutDuration: Account will be disabled for this amount of time when maxLoginAttempt is reached
```

Comment:
```suggestion
- `lockoutDuration`: Account will be disabled for this amount of time when `maxLoginAttempt` is reached
```

Reply:
Also, what values can this propertie take? seconds? milliseconds? It's not clear when only reading the docs.